recipes/eastl/all/conanfile.py

```diff
@@ -1,20 +1,35 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
 import os
 
-required_conan_version = ">=1.33.0"
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conan.tools.files import (
+    apply_conandata_patches,
+    copy,
+    export_conandata_patches,
+    get,
+    replace_in_file,
+)
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.52.0"
 
 
 class EastlConan(ConanFile):
     name = "eastl"
-    description = "EASTL stands for Electronic Arts Standard Template Library. " \
-                  "It is an extensive and robust implementation that has an " \
-                  "emphasis on high performance."
+    description = (
+        "EASTL stands for Electronic Arts Standard Template Library. "
+        "It is an extensive and robust implementation that has an "
+        "emphasis on high performance."
+    )
     topics = ("eastl", "stl", "high-performance")
     license = "BSD-3-Clause"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/electronicarts/EASTL"
 
+    package_type = "header-library"
```

Comment:
But it doesn't seem that this library is header-only, but a regular library. Am I missing something? https://github.com/electronicarts/EASTL/blob/c6141b94d763d2f536bddf40d6abaf878f7ab7ac/CMakeLists.txt#L25